pkg/controller/plan/adapter/ova/ovfparser.go

```diff
@@ -120,27 +120,35 @@ func GetFirmwareFromConfig(vmConfigXML string) (firmware string, err error) {
 	return BIOS, nil
 }
 
-func GetOperationSystemFromConfig(vmConfigXML string) (os string, err error) {
-	xmlConf, err := readConfFromXML(vmConfigXML)
+type InspectionOS struct {
```

Comment:
The xml output of `virt-v2v-inspector` is not strict ovf format, can we move the inspector xml parsing to a general pkg, instead of (re-)using the ova import package ?

Reply:
Done, ptal